composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^5.6|^7.0",
         "doctrine/dbal": ">=2.4.5,<3.0.x-dev",
-        "phpcr/phpcr": "~2.1.2",
+        "phpcr/phpcr": "~2.1.3",
```

Comment:
this means we are not compatible with 2.1.2 anymore. i think there is no need to increase the minimal version. `~` means at least this version, or any newer patch release of 2.1.